### DISCUSSION TITLE
lora fusion checkpoint

### DISCUSSION THREAD

**Discussion by s1cwz** [hako-mikan/sd-webui-supermerger#301] [Q&A]
I looked for a lot of videos on the Internet and couldn't find the problem
A lora dim32
A lora dim 64

A checkpoint

Fusion error report for help

![微信图片_20231202120156](https://github.com/hako-mikan/sd-webui-supermerger/assets/47632888/199034d7-ff68-46b3-ac60-ebf912bff82f)
![微信图片_20231202120207](https://github.com/hako-mikan/sd-webui-supermerger/assets/47632888/374717e0-f01b-432a-abf1-c367a0d8649e)

**Comment by hako-mikan** [comment]
Fixed in latest version.
